src/components/playback/mediasession.js

```diff
@@ -1,45 +1,26 @@
-define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], function (playbackManager, nowPlayingHelper, events, connectionManager) {
-    "use strict";
-
-    // no support for mediaSession
-    if (!navigator.mediaSession && !window.NativeShell) {
-        return;
-    }
+import playbackManager from 'playbackManager';
+import nowPlayingHelper from 'nowPlayingHelper';
+import events from 'events';
+import connectionManager from 'connectionManager';
+/* eslint-disable indent */
 
     // Reports media playback to the device for lock screen control
 
-    var currentPlayer;
-    var lastUpdateTime = 0;
-
-    function seriesImageUrl(item, options) {
+    let currentPlayer;
 
+    function seriesImageUrl(item, options = {}, type = options.type || 'Primary') {
         if (item.Type !== 'Episode') {
             return null;
-        }
-
-        options = options || {};
-        options.type = options.type || "Primary";
```

Comment:
this setting is missed, what if server or api client uses that?..

Reply:
@MrTimscampi I think this might be related to my review